easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-5.0.0-2023a.yml

```
@@ -4,3 +4,12 @@ easyconfigs:
         # see https://github.com/easybuilders/easybuild-easyconfigs/pull/22292
         # and https://github.com/easybuilders/easybuild-easyconfigs/pull/22646
         from-commit: b212c00fdc3983678037429719f1b210cb978b42
+  - MUSCLE-5.1.0-GCCcore-12.3.0.eb:
+      options:
+        # See https://github.com/easybuilders/easybuild-easyconfigs/pull/22780
+        from-commit: ff159780fbaa881fb442b0ee1b37cd9e3d981ca9
+  - bcgTree-1.2.1-foss-2023a.eb:
+      options:
+        # see https://github.com/easybuilders/easybuild-easyconfigs/pull/22707
+        # and https://github.com/easybuilders/easybuild-easyconfigs/pull/22540
```

The commit seems to be from PR 22781, so let's add that one here as well (no need to rebuild anything, since we're just adding a comment):
```suggestion
        # and https://github.com/easybuilders/easybuild-easyconfigs/pull/22540
        # and https://github.com/easybuilders/easybuild-easyconfigs/pull/22781
```